lib/log.js

```diff
@@ -3,11 +3,11 @@ import rfs from 'rotating-file-stream'
 import { basename, dirname } from 'node:path'
 import { paths } from './paths.js'
 
-export const formatLog = text =>
+export const formatLog = (text, date = new Date()) =>
   text
     .trimEnd()
     .split(/\n/g)
-    .map(line => `[${new Date().toLocaleTimeString()}] ${line}`)
+    .map(line => `[${date.toLocaleString()}] ${line}`)
```

Comment:
Depending on the locale, this new format can produce variable width. This will make logs more difficult to read.

```
[3/14/2023, 9:59:14 AM] INFO  Saturn Node is online and connected to 1 peers
[3/14/2023, 10:03:14 AM] INFO Saturn Node is online and connected to 15 peers
```

I don't have any better solution to offer. Since the width changes only infrequently, I guess it's not that a big deal.

FWIW, I get the following format on my machine:

```
[14/03/2023, 09:31:54]
```

Reply:
Yeah, I would like to fix this too, but don't see a solution that isn't really complex either